provider/scaleway/scaleway.go

```diff
@@ -65,7 +65,13 @@ func NewScalewayProvider(ctx context.Context, domainFilter endpoint.DomainFilter
 			defaultPageSize = 1000
 		}
 	}
-	p, _ := scw.MustLoadConfig().GetActiveProfile()
+
+	p := &scw.Profile{}
+	c, err := scw.LoadConfig()
+	if err == nil {
+		p, _ = c.GetActiveProfile()
```

Comment:
Why do you think that you can ignore the error?

Reply:
If the user sets environment variable we don't need to mount a secret and load the config file, the error can be ignored. If a config file can't be loaded and env vars are not set the scaleway client provider will return an error that is handled below.

Reply:
Then don't assign an error.
This is not good code, because it's not obvious: 
```
c, err := foo()
if err == nil {
  c.bar()
}
```
What would be better is to do this:

```
type foo struct{}
func (f *foo) bar() (*bar, error) {
  if f == nil {
    return nil, fmt.Errorf("foo is nil")
  }
...
}

func something() {
  c, _ := foo()
  b, _ := c.bar()
}
```